src/styles/card.scss

```diff
@@ -0,0 +1,155 @@
+.card {
+  box-sizing: border-box;
+  margin: 50px 0 46px var(--card-margin-left);
+  max-height: 408px;
+  width: var(--card-max-width);
+  background-color: #fff;
+  border: var(--card-border) solid #f3f3f3;
+  border-radius: var(--card-border-radius);
+}
+
+.card__sectionimage {
+  box-sizing: border-box;
+  width: 100%;
+  margin: var(--card-margin);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding-left: var(--min-section-gap);
+  padding-right: var(--min-section-gap);
+  padding-top: var(--card-header-top-gap);
+}
+
+.card__section {
+  display: block;
+  box-sizing: border-box;
+  width: 100%;
+}
+
+.card__image {
+  height: var(--card-image-height);
+  margin: var(--card-margin);
+  padding: var(--card-padding);
+}
+
+.card__title {
+  margin: var(--card-margin);
+  vertical-align: text-top;
+  padding-left: var(--min-section-gap);
+  padding-right: var(--min-section-gap);
+  padding-top: var(--card-padding-top);
+  font-size: var(--card-font-size);
+  font-weight: 500;
+  font-family: Roboto, sans-serif;
+  font-style: medium;
+  line-height: 18px;
+  color: var(--cardfont-color);
+}
+
+.card__description {
+  margin: 0;
+  padding: 0;
```

Comment:
You can set all the paddings directly in this shorthand property. Please fix all cases